PackageInfo.g

```diff
@@ -99,6 +166,7 @@ Persons := [
     Email         := "sal@cs.st-andrews.ac.uk",
     WWWHome       := "http://www-groups.dcs.st-and.ac.uk/~sal/",
     PostalAddress := Concatenation( [
+                       "Steve Linton\n",
```

Comment:
I’ll just say I think it’s weird and unnecessary to repeat the name at the beginning of the `PostalAddress` field. I’m all for it being consistent throughout the file, but if we’re going to choose a fixed way, I’d advocate always leaving out the name.

Reply:
I agree.

Personally I even consider the whole PostalAddress pointless.

Reply:
I'm happy to remove all Postal addresses, we just decide once. As some contributors had Postal Addresses and some didn't I added all of them. I'm happy to remove all of them. Any objections?